default.py

```diff
@@ -33,9 +33,12 @@ class Player(xbmc.Player):
     def __init__(self):
         xbmc.Player.__init__(self)
         log('Player - init')
-        self.mye = self._loginMyEpisodes()
-        if not self.mye.is_logged:
-            return
+        self.keep_going = True
```

Comment:
why do you need this variable ? It's the same info as the tests you do with login_attempts. You can only use login_attemps and define an options with the number of retries that could be changed by the users.